### PR TITLE
add rest API endpoints for indy tracking service

### PR DESCRIPTION
**Pull request by hjmodi**
* MMENG-3481

Signed-off-by: Harsh Modi <hmodi@redhat.com>

### CODE REVIEW

File: config/application.yaml
Comment (sswguo) [inline config/application.yaml:133]: FYI: this should be same with the existing keyspace for folo in cassandra.

File: src/main/java/org/commonjava/indy/service/tracking/jaxrs/AdminResource.java
Comment (sswguo) [inline src/main/java/org/commonjava/indy/service/tracking/jaxrs/AdminResource.java:40]: Since there is no specific implementation, looks good for me as long as the endpoints same with ones in monolith, to avoid the change in user side.

**Comment by ligangty** [comment]
Seems this is more like a skeleton setup, so I don't have opinions to reject this PR.

File: config/application.yaml
Comment (hjmodi) [reply config/application.yaml]: updated

File: src/main/java/org/commonjava/indy/service/tracking/jaxrs/AdminResource.java
Comment (hjmodi) [reply src/main/java/org/commonjava/indy/service/tracking/jaxrs/AdminResource.java:40]: yes, I've made sure it's same

**Comment by sswguo** [comment]
LGTM.

**Comment by hjmodi** [comment]
@ligangty @sswguo Thanks for providing your approval. Can you please also approve the review in GitHub system? Due to branch protection rules I have set it requires at least one approval before allowing me to merge.

**Review by ligangty — APPROVED** [review]

**Pull request merged**
